lib/Dashboard/TalkWidget.php

```diff
@@ -170,6 +174,53 @@ public function getItems(string $userId, ?string $since = null, int $limit = 7):
 		return $result;
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	public function getItemsV2(string $userId, ?string $since = null, int $limit = 7): WidgetItems {
+		$sessionIds = $this->session->getAllActiveSessions();
+		$allRooms = $this->manager->getRoomsForUser($userId, $sessionIds, true);
```

Comment:
You are actually not using the sessions, so we can also skip fetching them.

```suggestion
		$allRooms = $this->manager->getRoomsForUser($userId, [], true);
```